package.json

```diff
@@ -97,6 +97,6 @@
   "snyk": true,
   "dependencies": {
     "@uswds/compile": "1.2.0",
-    "@uswds/uswds": "3.9.0"
+    "@uswds/uswds": "github:uswds/uswds#develop"
```

Comment:
**chore**: Will need to update to `3.10.0`

Reply:
Done in https://github.com/uswds/uswds-site/pull/2944/commits/e5109d9f8ff316930e3c46107e37de8681a9813a